Documentation/upgrade.md

```diff
@@ -116,7 +117,8 @@ kubectl -n rook get pod ${MON0_POD_NAME} -o jsonpath='{.spec.containers[0].image
 ### All Pods Status and Version
 The status and version of all Rook pods can be collected all at once with the following commands:
 ```bash
-kubectl get pod -l app=rook-operator -o jsonpath='{range .items[*]}{.metadata.name}{"\n\t"}{.status.phase}{"\t"}{.spec.containers[0].image}{"\n"}{end}'
+kubectl -n rook-system get pod -l app=rook-operator -o jsonpath='{range .items[*]}{.metadata.name}{"\n\t"}{.status.phase}{"\t"}{.spec.containers[0].image}{"\n"}{end}'
```

Comment:
same here, no need to do 2 different commands each with their own `-l`.  since everything in the rook-system namespace is for rook, it can just be a single command that works on the whole rook-system namespace and does not specify any labels with `-l`.

Reply:
done